keyboards/nightmare/info.json

```diff
@@ -0,0 +1,264 @@
+{
+    "keyboard_name": "Nightmare",
+    "url": "https://keyhive.xyz/shop/nightmare-pcb-and-case",
+    "maintainer": "cfbender",
+    "width": 15.25,
+    "height": 4,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {
```

Comment:
Your [layout macro](https://github.com/qmk/qmk_firmware/blob/746d92e09eae58b76410df7024184e8cc34958e5/keyboards/nightmare/nightmare.h#L28-L33) has 52 keys but this file only describes 49. This JSON tree needs to have one object for each key represented by the layout macro.

Reply:
This may arise from how I wired it. The layout macro has an extra 3 keys that cover the split space option. Would it be better if I were to put those in a different layout macro? This is my first time doing multiple layouts on a board.

Reply:
Yes, an additional layout macro in `keyboards/nightmare/nightmare.h` and another JSON tree under `layouts` in `keyboards/nightmare/info.json` would be appropriate.

[keyboards/gh60/satan/info.json](https://github.com/qmk/qmk_firmware/blob/37b6a2abbd96eaf3d6d724ac09c789d54a67d962/keyboards/gh60/satan/info.json) is a good example of how to set up multiple layouts in an info.json file.

Reply:
Awesome, thanks @noroadsleft. Hopefully have this taken care of on the latest commit

Reply:
@noroads, this look good now?